docs/users/developer-tools.md

```diff
@@ -24,17 +26,19 @@ Please note this will not intercept emails if your application is configured to
 MailHog provides several [configuration options](https://github.com/mailhog/MailHog/blob/master/docs/CONFIG.md). If you need to alter its configuration, you can do so by adding the desired environment variable to the `environment` section for the web container in the `.ddev/docker-compose.yaml` for your project.
 
 ## Using Development Tools on the Host Machine
+
 It is possible in many cases to use development tools installed on your host machine on a project provisioned by ddev. Tools that interact with files and require no database connection, such as Git or Composer, can be run from the host machine against the code base for a ddev project with no additional configuration necessary.
 
 ### Database Connections from the Host
-If you need to connect to the database of your project from the host machine, run `ddev describe` to retrieve the Database connection information. The last line of the database credentials will provide your host connection info, similar to this:
+
+If you need to connect to the database of your project from the host machine, run `ddev describe` to retrieve the database connection information. The last line of the database credentials will provide your host connection info, similar to this:
 
 ```
 To connect to mysql from your host machine, use port 32838 on 127.0.0.1
 For example: mysql --host 127.0.0.1 --port 32838
 ```
 
-The port referenced is unique per running project, and randomly chosen from available ports on your system when `ddev start` is ran.
+The port referenced is unique per running project, and randomly chosen from available ports on your system when you run `ddev start`.
```

Comment:
Can't believe that "is ran" had survived until now.  I have killed so many of those!

Reply:
I left passive, vague language here :/ 

What is referencing the port? ddev, right?

> The port referenced is unique per running project, and randomly chosen from available ports on your system when you run `ddev start`.